packages/reactive-element/src/reactive-element.ts

```diff
@@ -1390,6 +1390,35 @@ export abstract class ReactiveElement
           );
         }
       }
+      // Mixin instance properties once, if they exist.
+      if (this.__instanceProperties) {
+        // TODO (justinfagnani): should we use the stored value? Could a new value
+        // have been set since we stored the own property value?
+        for (const [p, value] of this.__instanceProperties) {
+          this[p as keyof this] = value as this[keyof this];
+        }
+        this.__instanceProperties = undefined;
+      }
+      // Trigger initial value reflection and populate the initial
+      // changedProperties map, but only for the case of experimental
+      // decorators on accessors, which will not have already populated the
+      // changedProperties map. We can't know if these accessors had
+      // initializers, so we just set them anyway - a difference from
+      // experimental decorators on fields and standard decorators on
+      // auto-accessors.
+      const elementProperties = (this.constructor as typeof ReactiveElement)
+        .elementProperties;
+      if (elementProperties.size > 0) {
```

Comment:
Ah, took me a sec to notice that this is still in the `!this.hasUpdated` condition